bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/api/Car.java

```diff
@@ -105,6 +116,9 @@ public void setBatteryStatus(JsonObject responseJson) {
                 if (attributes.get("chargingRemainingTime") != null) {
                     chargingRemainingTime = attributes.get("chargingRemainingTime").getAsInt();
                 }
+                if (attributes.get("timestamp") != null) {
+                    batteryStatusUpdated = ZonedDateTime.parse(attributes.get("timestamp").getAsString());
```

Comment:
This can throw [DateTimeParseException](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeParseException.html) if the format is not as expected.

Reply:
I have pushed a try / catch for this.